src/web_app_skeleton/script/setup.ecr

```diff
@@ -56,12 +56,12 @@ printf "\n▸ Checking that postgres is installed\n"
 check_postgres | indent
 printf "✔ Done\n" | indent
 
+printf "\n▸ Creating the database\n"
+lucky db.create | indent
```

Comment:
This  should work since we check that Postgres `createdb` is installed. Still possible that Postgres is not running and `db.create` fails, but that is better than  verify_connection failing every time on first run of script/setup